### DISCUSSION TITLE
Missing Ancients at Final Level

### DISCUSSION THREAD

**Discussion by GitoGitoto** [FakeFishGames/Barotrauma#15433] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I made it to the end game, upon encountering Jove, the Ancients are missing. He's lonely for Christmas :(

### Reproduction steps

![20241222155350_1](https://github.com/user-attachments/assets/03bfc38c-aae8-43f6-a38e-325550ce3b25)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately couldn't find a way to reproduce the issue, the ancients do seem to spawn as expected.
![image](https://github.com/user-attachments/assets/9c1ce322-e397-47bf-9b0a-36e57f106d14)

Are you using some kind of mods? If you still have the save (or a backup) available, would it be possible to send it here? Thanks and sorry for the inconvenience!

**Reply by GitoGitoto** [reply]
Sorry for late response, I have no mods in use. Saldy, I think the backup for this level is no longer available. Thanks for the response!
